src/transformers/models/vit_mae/modeling_vit_mae.py

```diff
@@ -242,18 +245,21 @@ def initialize_weights(self):
         # timm's trunc_normal_(std=.02) is effectively normal_(std=0.02) as cutoff is too big (2.)
         torch.nn.init.normal_(self.cls_token, std=self.config.initializer_range)
 
-    def random_masking(self, sequence):
+    def random_masking(self, sequence, noise=None):
         """
         Perform per-sample random masking by per-sample shuffling. Per-sample shuffling is done by argsort random
         noise.
 
         Args:
             sequence (`torch.LongTensor` of shape `(batch_size, sequence_length, dim)`)
+            noise (`tf.Tensor` of shape `(1, sequence_length)`) which is only used
+                for testing purposes to control randomness
```

Comment:
Here should be `torch.FloatTensor`. The shape is still better to stated as `(batch_size, sequence_length)`.
Although we introduce it for testing purpose, I would suggest to say:

... which is **mainly** used for testing purposes to control randomness and maintain the reproducibility

Reply:
Will take care of that.